client/js/lounge.js

```diff
@@ -594,7 +594,8 @@ $(function() {
 			}
 		});
 	});
-	if ($("body").hasClass("public")) {
+
+	if ($("body").hasClass("public") && window.location.hash === "#connect") {
```

Comment:
Can we support `#channels` in public mode instead?

Reply:
Can you elaborate?

Reply:
oh, if public mode is trying to connect to channels, populate the channels textbox? Then you're assuming that the user also knows the network, etc. Also, we should probably encourage them to use https://github.com/thelounge/lounge/pull/674

Reply:
Yeah, that feels like 2 ways to solve the 1 issue.